cmd/osbuild-worker/main.go

```diff
@@ -79,7 +79,7 @@ func RunJob(job *worker.Job, uploadFunc func(*worker.Job, io.Reader) error) (*co
 
 	for _, t := range job.Targets {
 		switch options := t.Options.(type) {
-		case *target.LocalTargetOptions:
+		case *target.WeldrTargetOptions:
```

Comment:
As discussed, we may want to simply drop the target altogether, and make it a high-level concept in the Job. This would make it just an artefact of a job and not weldr specific.

Reply:
Agreed. I dropped the rename to `org.osbuild.weldr` and added the new fields to the local target. If we rework this in the future, there's no point to break backwards compat twice.